src/analyses/local_bitvector_analysis.cpp

```diff
@@ -242,8 +242,8 @@ void local_bitvector_analysist::build()
   if(cfg.nodes.empty())
     return;
 
-  work_queuet work_queue;
```

Comment:
This one has a friend who would also like to become a set:

https://github.com/diffblue/cbmc/blob/eeddb3feb756be04e08709d9989d8ed2c36d793c/src/analyses/local_may_alias.h#L61

Reply:
Good call, I'll do so in a follow-up PR.